samples/panic_recover.go

```diff
@@ -0,0 +1,23 @@
+// panic_recover
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	n := panicTest()
```

Comment:
this is a little odd that you are taking the return value, but the panic path does not return a value (so you're getting the zero value for `int`). You then print it out as though it has some significance. Since in this example never actually returns a value, how about we just ignore that at the call site and print something else.

https://goplay.space/#6k1IKwJZkhN

I changed "failed" to "panicking" and printed "recovered from:" to show where each bit of text comes from, and then just printed "Still alive" to show the program was not aborted.